webapp/channels/src/components/root/__snapshots__/root.test.tsx.snap

```diff
@@ -0,0 +1,136 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/Root Routes Should mount public product routes 1`] = `
+<RootProvider>
+  <Switch>
+    <Route
+      component={[Function]}
+      path="/error"
+    />
+    <HFRoute
+      component={[Function]}
+      path="/login"
+    />
+    <HFRoute
+      component={[Function]}
+      path="/access_problem"
+    />
+    <HFTRoute
+      component={[Function]}
+      path="/reset_password"
+    />
+    <HFTRoute
+      component={[Function]}
+      path="/reset_password_complete"
+    />
+    <HFRoute
+      component={[Function]}
+      path="/signup_user_complete"
+    />
+    <HFRoute
+      component={[Function]}
+      path="/should_verify_email"
+    />
+    <HFRoute
+      component={[Function]}
+      path="/do_verify_email"
+    />
+    <HFTRoute
+      component={[Function]}
+      path="/claim"
+    />
+    <HFTRoute
+      component={[Function]}
+      path="/help"
+    />
+    <LoggedInRoute
+      component={[Function]}
+      path="/terms_of_service"
+    />
+    <Route
+      component={[Function]}
+      path="/landing"
+    />
+    <Route
+      path="/admin_console"
+    >
+      <Switch>
+        <LoggedInRoute
+          component={[Function]}
+          path="/admin_console"
+          theme={Object {}}
+        />
+        <Connect(RootRedirect) />
+      </Switch>
+    </Route>
+    <LoggedInHFTRoute
+      component={[Function]}
+      path="/select_team"
+    />
+    <LoggedInHFTRoute
+      component={[Function]}
+      path="/oauth/authorize"
+    />
+    <LoggedInHFTRoute
+      component={[Function]}
+      path="/create_team"
+    />
+    <LoggedInRoute
+      component={[Function]}
+      path="/mfa"
+    />
+    <LoggedInRoute
+      component={[Function]}
+      path="/preparing-workspace"
+    />
+    <Redirect
+      from="/_redirect/integrations/:subpath*"
+      to="//integrations/:subpath*"
```

Comment:
This seems off with the double slashes

Reply:
It seems to be normal for jest snapshots, as we have more instances of it across the codebase

Reply:
This was due to TeamId being empty ("") in the test data, I updated the test and snapshot.